client/src/common/client.ts

```diff
@@ -689,6 +706,13 @@ export abstract class BaseLanguageClient implements FeatureClient<Middleware, La
 		if (this._didChangeTextDocumentFeature!.syncKind === TextDocumentSyncKind.Full) {
 			await this.sendPendingFullTextDocumentChanges(connection);
 		}
+
+		const generalMiddleware = this._clientOptions.middleware?.sendRequest;
```

Comment:
To make this consistent with other implementation can we do the following:

- rename to _sendrequest
- then use a ? operation like here: https://insiders.vscode.dev/github/microsoft/vscode-languageserver-node/blob/d859bb14d1bcb3923eecaf0ef587e55c48502ccc/client/src/common/completion.ts#L130

Reply:
I meant _sendRequest

Reply:
Done!